app/controllers/users_controller.rb

```diff
@@ -16,9 +17,10 @@ def create
   end
 
   def show
-    @user = User.find_by(params[:id])
-    @todos = @user.todos
-    @todo = Todo.new
+    result = ShowUser.call(id: params[:id])
```

Comment:
@Enriikke so here's one that really just seems to obfuscate our intensions. I do like having the `show_user.rb` file in our `interactors` folder though. Seems consistent.